pom.xml

```diff
@@ -289,22 +272,68 @@
         </plugin>
         <plugin>
           <artifactId>maven-site-plugin</artifactId>
-          <version>3.9.1</version>
+          <version>3.12.1</version>
         </plugin>
         <plugin>
           <groupId>com.diffplug.spotless</groupId>
           <artifactId>spotless-maven-plugin</artifactId>
-          <version>2.17.3</version>
+          <version>2.25.0</version>
+        </plugin>
+        <plugin>
+          <groupId>org.codehaus.mojo</groupId>
+          <artifactId>build-helper-maven-plugin</artifactId>
+          <version>3.3.0</version>
         </plugin>
       </plugins>
     </pluginManagement>
+    <plugins>
+      <plugin>
+        <!-- Must have it here as pluginMgmt would NOT override since parent POM defined it here as well -->
```

Comment:
I would rather recommend removing the no longer maintained parent oss-parent (https://central.sonatype.org/publish/publish-maven/#deprecated-oss-parent).

Reply:
Similar PR in mojos... https://github.com/eclipse/sisu.mojos/pull/7 maybe we need some sisu parent instead?

Reply:
Let us track the introduction of a new parent pom.xml separately.

Reply:
Let us track this separately from this PR.